### ISSUE TITLE
Restapi caching granularity

### ISSUE THREAD

**Issue by cekk** [plone/Products.CMFPlone#4017]
Now all restapi routes are configured with terseCaching.
restapi routes can have different needs so we could be able to assign different rulesets to them.

For example content GET can be cached for more time than high dynamic endpoints like @search or @querystring-search

Proposal:
- In plone.restapi change [content GET](https://github.com/plone/plone.restapi/blob/main/src/plone/restapi/services/content/configure.zcml#L21)to use "plone.content.itemView" ruleset (this allows us to better tweak caching parameters)
- Are there other endpoints that need a different cache? @navigation? 
- Could it be a good idea adding a third default preset in p.a.caching defaults, for sites with volto? Or maybe set it directly in plone.restapi or plone.volto

**Comment by davisagli** [comment]
> For example content GET can be cached for more time than high dynamic endpoints like @search or @querystring-search

It depends... if there are API expanders enabled, then the content GET might also include data from other content items (e.g. navigation & breadcrumbs components)

**Comment by cekk** [comment]
> > For example content GET can be cached for more time than high dynamic endpoints like @search or @querystring-search
> 
> It depends... if there are API expanders enabled, then the content GET might also include data from other content items (e.g. navigation & breadcrumbs components)

yes, you're right, but having a separate caching ruleset allows us to potentially tweak settings only for contents.
If you don't need or don't want (because as you said there are some dynamic infos in expanders), you could stay with terse.

**Comment by folix-01** [comment]
We decided that there is no necessity to have the separated profiles. But we just give the possibility to manage granularly the caching rules for RestAPI Services giving the priority in caching rules negotiation if the service name is used in the caching operation ruleset.

Prs: https://github.com/plone/plone.app.caching/pull/143 , https://github.com/plone/plone.rest/pull/183
